src/journal/journalctl.c

```diff
@@ -509,6 +510,11 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case 'o':
+                        if (streq(optarg, "help")) {
+                                DUMP_STRING_TABLE(output_mode, OutputMode, _OUTPUT_MODE_MAX);
+                                return 0;
+                        }
+
```

Comment:
Heh, I was thinking about implementing this for `journalctl -o` the other day. Great to see this.